lib/streamlit/elements/exception.py

```diff
@@ -149,28 +149,27 @@ def marshall(
     if is_uncaught_app_exception:
         show_error_details = config.get_option("client.showErrorDetails")
 
-        # Options for show error details config.
-        FULL = "full"
-        STACKTRACE = "stacktrace"
-        TYPE = "type"
         # Config options can be set from several places including the command-line and
         # the user's script. Legacy config options (true/false) will have type string when set via
         # command-line and bool when set via user script (e.g. st.set_option("client.showErrorDetails", False)).
         TRUE_VARIATIONS = ["true", "True", True]
         FALSE_VARIATIONS = ["false", "False", False]
```

Comment:
nit: we could think about moving these to the `ShowErrorDetailsConfigOptions` enum as well as a function, like

```python
class ShowErrorDetailsConfigOptions(str, Enum):
     """Valid options for the "client.showErrorDetails" config."""
 
     FULL = "full"
     STACKTRACE = "stacktrace"
     TYPE = "type"
     NONE = "none"
     
    def is_true_variation(val: str | bool):
         return val in ["true", "True", True]

    def is_false_variation(val: str | bool):
       return val in ["false", "False", True]
```

which can then be used as `ShowErrorDetailsConfigOptions.is_true_variation(show_error_details)`